blocks/DataOutput/DataOutput.js

```diff
@@ -1,12 +1,13 @@
-import { Block } from '../../abstract/Block.js';
+import { UploaderBlock } from '../../abstract/UploaderBlock.js';
 import { uploadFileGroup } from '../../submodules/upload-client/upload-client.js';
 
 /** @typedef {import('../../submodules/upload-client/upload-client.js').UploadcareFile[]} FileList} */
 
-export class DataOutput extends Block {
+export class DataOutput extends UploaderBlock {
```

Comment:
A bit strange thing that DataOutput extends UploaderBlock which extends ActivityBlock because DataOutput isn't activity .